lib/xwax/pitch.h

```diff
@@ -1,71 +1,213 @@
+
 /*
- * Copyright (C) 2021 Mark Hills <mark@xwax.org>
- *
- * This file is part of "xwax".
- *
- * "xwax" is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License, version 3 as
- * published by the Free Software Foundation.
- *
- * "xwax" is distributed in the hope that it will be useful, but
- * WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * General Public License for more details.
+ * Kalman filter replacement for alpha-beta pitch estimator
```

Comment:
This comment is no longer valid after merging. 
Maybe you can reference the version number up to the old filter was uses it such.